content/blog/2025/03/kubectl-1-22-1-patch-release.md

```diff
@@ -0,0 +1,23 @@
+---
+title: kubectl 1.22.1 patch release
```

Comment:
```suggestion
title: Kubectl 1.22.1 patch release
```

Reply:
I just noticed that should be kwctl. I've fixed it everywhere